json_schema_for_humans/generate.py

```diff
@@ -121,6 +121,8 @@ def _copy_additional_file_to_target(
     source_file_path = source_directory / file_to_copy
     if not source_file_path.exists():
         return
+    parent = (target_directory / file_to_copy).parent
+    parent.mkdir(exist_ok=True, parents=True)
```

Comment:
Wouldn't `parent`  always be the same as `target_directory` in that context?

It's ok to make sure the target directory exists though

Reply:
I added this because I put the bundled files in subdirectories `css`, `font` and `js` that will be within `target_directory`.